src/main/java/org/apache/pulsar/logstash/inputs/Pulsar.java

```diff
@@ -105,8 +105,14 @@ public class Pulsar implements Input {
     private static final PluginConfigSpec<String> CONFIG_TLS_TRUST_STORE_PASSWORD =
             PluginConfigSpec.stringSetting("tls_trust_store_password","");
 
-    private static final PluginConfigSpec<String> CONFIG_AUTH_PLUGIN_CLASS_NAME =
-            PluginConfigSpec.stringSetting("auth_plugin_class_name",authPluginClassName);
+    private static final PluginConfigSpec<Boolean> CONFIG_ENABLE_AUTH =
+            PluginConfigSpec.booleanSetting("enable_auth",false);
+
+    private static final PluginConfigSpec<String> CONFIG_AUTH_PLUGIN =
```

Comment:
I'm sorry for making an incorrect review. This change breaks the` auth_plugin_class_name` config. Please revert this.